src/lib/custom-resources/cdk-logs-metric-filter/runtime/src/index.ts

```diff
@@ -71,7 +71,7 @@ async function onCreateOrUpdate(event: CloudFormationCustomResourceEvent) {
         .promise(),
     );
   } else {
-    console.warn(`Did not find LogGroup ${logGroupName}`);
+    throw `Did not find LogGroup ${logGroupName}`;
```

Comment:
Believe this should be formatted as follows:
throw new Error(`Did not find LogGroup ${logGroupName}`);

Reply:
Thanks!